test/e2e/tests/test_embedding_inference.py

```
@@ -0,0 +1,245 @@
+        model = _get_cr("maasmodelref", EMBEDDING_MODEL_REF, namespace=MODEL_NAMESPACE)
+        if not model:
+            pytest.skip(f"MaaSModelRef {EMBEDDING_MODEL_REF} not deployed")
+
+        oc_token = _get_cluster_token()
+        # Mint a key bound to any available subscription (won't match this model)
+        try:
+            api_key = _create_api_key(
+                oc_token,
+                name=f"e2e-emb-deny-{uuid.uuid4().hex[:8]}",
+            )
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)